lib/electoral_history.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+require 'scraped'
+
+class ElectoralHistory < Scraped::HTML
+  def to_h
```

Comment:
clobbering the base `to_h` is almost certainly a very bad idea. Doing this essentially negates it being a `Scraped::HTML` subclass. Can you not implement this using `field`s as normal?